tools/bin/load_test/load_test_utils.sh

```diff
@@ -0,0 +1,49 @@
+<<comment
+This file contains common util functions for use in load testing scripts.
+comment
+
+echo "Loading utils from $0"
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;34m'
+CLEAR='\033[0m'
+
```

Comment:
for my knowledge, what is this for? is it better logging?

Reply:
yeah - like `${CLEAR}-P <port>
    ${GREEN}Specify the port for the Airbyte server.` assigns the instruction words with color